src/site/content/en/blog/indexeddb/index.md

```diff
@@ -454,144 +620,184 @@ function searchItems(lower, upper) {
-
+The code first gets the values for the limits and checks if the limits exist. The next block of code
+decides which method to use to limit the range based on the values. In the database interaction, you
+open the object store on the transaction as usual, then you open the `"price"` index on the object
+store. The `"price"` index allows you to search for the items by price. You open a cursor on the
+index and pass in the range. The cursor now returns a promise representing the first object in the
+range, or `undefined` if there is no data within the range. The `cursor.continue()` method returns a
+cursor representing the next object and so on through the loop until you reach the end of the range.
 
 ## Using database versioning {: #versioning }
 
-When we call `idb.open`, we can specify the database version number in the second parameter. If this version number is greater than the version of the existing database, the upgrade callback executes, allowing us to add object stores and indexes to the database.
+When you call `idb.open()`, you can specify the database version number in the second parameter. If
+this version number is greater than the version of the existing database, the upgrade callback
+executes, allowing you to add object stores and indexes to the database.
 
-{% Aside %}
-The browser throws an error if we try to create object stores or indexes that already exist in the database. We can wrap the calls to `createObjectStore` in `if` statements checking if the object store already exists using `upgradeDb.objectStoreNames.contains('objectStoreName')`. We can also use a `switc` statement on the `oldVersion` property as in the next example.
+{% Aside %} The browser throws an error if you try to create object stores or indexes that already
+exist in the database. You can wrap the calls to `createObjectStore` in `if` statements checking if
+the object store already exists using `upgradeDb.objectStoreNames.contains('objectStoreName')`. You
+can also use a `switch` statement on the `oldVersion` property as in the next example.
 {% endAside %}
 
-
-The UpgradeDB object has a special `oldVersion` property, which indicates the version number of the database existing in the browser. We can pass this version number into a `switch` statement to execute blocks of code inside the upgrade callback based on the existing database version number. Let's look at an example:
+The `UpgradeDB` object has a special `oldVersion` property, which indicates the version number of
+the database existing in the browser. You can pass this version number into a `switch` statement to
```

Comment:
>existing in the browser

The version number of an existing database already in the browswer? Or are you trying to convey something else?

Reply:
Yes, the version number of an existing database already in the browser.